applications/adcirc-frontera/app.json

```diff
@@ -0,0 +1,92 @@
+{
+    "id": "adcirc",
+    "version": "55.01",
+    "description": "ADCIRC, ADvanced CIRCulation, is a system of computer programs for solving time-dependent, free surface circulation and transport problems in two and three dimensions. These programs utilize the finite element method in space allowing the use of highly flexible and unstructured grids.",
+    "owner": "${apiUserId}",
+    "enabled": true,
+    "runtime": "ZIP",
+    "runtimeVersion": null,
+    "runtimeOptions": null,
+    "containerImage": "tapis://cloud.data/corral/tacc/aci/CEP/applications/v3/adcirc/v55/adcirc.zip",
```

Comment:
I get permission denied for this file when I try to run the app published in dev.cep